network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/featured_blogs.html

```diff
@@ -0,0 +1,64 @@
+{% load wagtailimages_tags %}
+
+{% if page.featured_blogs.all.count > 0 %}
+{% with page.featured_blogs.all|dictsortreversed:"blog.first_published_at" as featured_blogs %}
+<div class="row">
+  <div class="col mb-4">
+    <hr class="mb-4 hr-emphasis mt-0">
+    <h4 class="h4-heading">Latest news</h4>
+  </div>
+</div>
+<div class="row mb-3">
+  {% for blog in featured_blogs %}
+  {% if forloop.counter is 1 %}
+  <div class="col-md-6 mb-0 pb-4">
+    <div>
+      {% image blog.blog.specific.get_meta_image width-540 %}
+    </div>
+    <div class="row">
```

Comment:
How much of this template is shared with the blog card template? Do we want to make those shared code, or keep them separate?

Reply:
They _could_ share the same code but it will complicate the logics in `blog-card.html` further as their DOM element types and class names are actually quite different. I'd keep them separate to keep things clean.